src/cmd_all/Cargo.toml

```diff
@@ -15,7 +16,7 @@ risingwave_frontend = { path = "../frontend" }
 risingwave_logging = { path = "../utils/logging" }
 risingwave_meta = { path = "../meta" }
 tikv-jemallocator = "0.4"
-tokio = { version = "1", features = [
+tokio = { version = "0.2.0-alpha.2", package = "madsim-tokio", features = [
```

Comment:
Is it possible to use madsim in a less intrusive way? I mean we may only enable madsim in deterministic tests and in typical situations we still use the `tokio` crate from crates.io directly.

Maybe something like this.
```toml
[target.'cfg(features = "sim")'.dependencies] # sim enabled
tokio = { version = "0.2.0-alpha.2", package = "madsim-tokio"}
[target.'cfg(not(features = "sim"))'.dependencies] # sim disabled
tokio = "1"
```

Reply:
In fact, `madsim-tokio` does the same thing internally. The only difference is where we put the `sim` feature. If we can trust `madsim-tokio`, I think it's acceptable to replace `tokio` directly. Maybe not looks good, but it works. 🙃